rootfs/scheduler/__init__.py

```diff
@@ -568,42 +583,53 @@ def run(self, namespace, name, image, entrypoint, command, **kwargs):
         if unhealthy(response.status_code):
             error(response, 'create Pod in Namespace "{}"', namespace)
 
-        data = ''
-        duration = 30
-        iteration = 1
-        while (iteration < duration):
-            try:
+        labels = {
+            'app': namespace,
+            'type': 'run',
+            'version': kwargs.get('version'),
```

Comment:
No, `run` in `App` defines it as `version` (https://github.com/deis/controller/blob/master/rootfs/api/models/app.py#L582)